drivers/watchdog/wdt_cmsdk_apb.c

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define DT_DRV_COMPAT arm_cmsdk_watchdog
```

Comment:
Did these changes sneak in by mistake? They are not MCUX specific.

Reply:
Yeah, will remove.